server/index.js

```diff
@@ -81,6 +70,21 @@ module.exports = (_config = {}) => {
   const bcoinRouter = require('./bcoinRouter');
   const socketHandler = require('./bcoinSocket');
 
+  // get bpanel config
+  const bpanelConfig = path.resolve(os.homedir(), '.bpanel/.bpanel.js');
```

Comment:
Would it be possible to allow for multiple config files? One for each network doesn't seem general enough because people could have different clusters on the same network

Reply:
That's a good call. Right now, as is, I'm not sure, given current construction, how that would/should look, though I'm sure we could come up with something. My feeling is that the best way to manage different bPanel instances on the same server would be to allow for the top level directory to be customized. So each one only has one top `config.js` file, but you could have `.bpanel-1/config.js`, `.bpanel-2/config.js`, etc. This is kind of how bcfg works actually, via the module argument. 

Alternatively it could be done in the same way that bcoin organizes networks via bcfg, via the `suffix` option

Reply:
We already have `~/.bpanel/clients` and we can at runtime select which clients to use. I'm not sure if I like coupling clients and bpanel config, but one solution is to use the same argument that is passed to select the client and use it to select a bpanel config

Reply:
Yeah, I don't think they should be coupled as they are two different things. You could have one bpanel setup (with your chosen plugins, theme, etc.) which is set in the bpanel config and from there you will likely connect to a range of different nodes which is managed via clients. Also, changing the pluginsConfig requires a rebuild and an npm install which can take some time. When you switch clients, the changes are available almost immediately (server just needs to restart)